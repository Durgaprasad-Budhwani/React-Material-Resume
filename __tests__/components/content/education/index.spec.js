/**
 * Created by durgaprasad on 10/10/17.
 */
/* global expect,it */
import React from 'react';
import Education from '../../../../src/components/content/education';
import renderer from 'react-test-renderer';

it('renders education', () => {
  const tree = renderer.create(
    <Education
    />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});