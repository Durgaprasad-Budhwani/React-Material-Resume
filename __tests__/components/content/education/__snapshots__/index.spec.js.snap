// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Educations /> renders <EducationContainer /> 1`] = `
<div
  className="MuiGrid-typeContainer-5 MuiGrid-direction-xs-column-7 Educations-container-1"
>
  <span
    className="SectionTitle-sectionTitle-94"
  >
    Education
  </span>
  <div
    style={
      Object {
        "position": "relative",
      }
    }
  >
    <div
      className="Educations-clear-3"
    >
      <div
        className="TimelineBar-bar-95 Educations-barClass-4"
      />
    </div>
  </div>
</div>
`;

exports[`<Educations /> renders <Educations /> 1`] = `
<EventContainer
  alignment="RIGHT"
  classes={
    Object {
      "arrow": "EventContainer-arrow-105",
      "arrowLeft": "EventContainer-arrowLeft-107",
      "arrowRight": "EventContainer-arrowRight-106",
      "bigScreenBox": "EventContainer-bigScreenBox-101",
      "boxLeft": "EventContainer-boxLeft-102",
      "boxRight": "EventContainer-boxRight-103",
      "date": "EventContainer-date-100",
      "dot": "EventContainer-dot-104",
      "dotLeft": "EventContainer-dotLeft-109",
      "dotRight": "EventContainer-dotRight-108",
      "paper": "EventContainer-paper-97",
      "paperLeft": "EventContainer-paperLeft-98",
      "paperRight": "EventContainer-paperRight-99",
      "smallScreenBox": "EventContainer-smallScreenBox-96",
    }
  }
>
  <withStyles(Course)
    name="description"
    university="college"
    year="year"
  />
</EventContainer>
`;
